cmd/erasure.go

```diff
@@ -28,13 +28,17 @@ import (
 	"sync"
 	"time"
 
+	"github.com/dustin/go-humanize"
 	"github.com/minio/madmin-go/v3"
 	"github.com/minio/minio/internal/dsync"
 	xioutil "github.com/minio/minio/internal/ioutil"
 	"github.com/minio/minio/internal/logger"
 	"github.com/minio/pkg/v2/sync/errgroup"
 )
 
+// bigFileThreshold is the point where we add readahead to put operations.
+const bigFileThreshold = 64 * humanize.MiByte
```

Comment:
Readahead is nice, but also rather expensive. Could you share the numbers you are seeing by halving?